src/components/modals/share-modal/index.tsx

```diff
@@ -68,7 +90,7 @@ const ShareModal = ({ show, handleModal }: Props) => {
                 item
                   ? `Did you check the ${item.name}'s NFT?. Check it out!:`
                   : 'Check Out *Leda MarketPlace:* '
-              } *${window.location.href}*`}
+              } *${itemId ? `${window.location.origin}/item/${itemId}` : window.location.href}*`}
```

Comment:
I'd suggest to use the nextjs router here. Otherwise, you will cause a page refresh and delete redux state.

Reply:
Hmm, I guess I can't make external redirection with Next Router